tests/commit-lint.test.ts

```diff
@@ -158,3 +159,44 @@ blah http://www.github.com\n\nSigned-off-by: x`;
     }
 
 });
+
+if (process.env.gggdev) {
+    test("verify against existing commits", async () => {
+        const commit = {
+            author: {
+                email: "ggg@example.com",
+                login: "ggg",
+                name: "e. e. cummings",
+            },
+            commit: "BAD1FEEDBEEF",
+            committer: {
+                email: "ggg@example.com",
+                login: "ggg",
+                name: "e. e. cummings",
+            },
+            message: "Message has no description",
+            parentCount: 1,
+        };
+
+        const gitDir = "../git";
+        const patches = await git(["log",
+                                   "-300",
+                                   "--no-merges",
+                                   "--pretty=format:\"%h\""],
+                                  { workDir: gitDir });
```

Comment:
Like, look at the output of `git rev-list --since=6.months.ago --parents upstream/pu` line by line. For each line containing exactly three commit IDs `<merge> <first-parent> <second-parent>`, verify that the commit range `<first-parent>..<second-parent>` contains no merge commits (e.g. by running it through `git rev-list --parents` and looking for lines with two spaces) and if they don't, lint the range.

Reply:
Come to think of it, this might not be a good candidate for the test suite after all, but more for manual testing via `scripts/manual-linting.ts`?

Reply:
> I wonder whether it would make more sense to simply use past examples

That was the purpose of this.  I had not considered how git changes are done so just grabbing non-merge commits off the current branch does not get everything.  It was a reaction to seeing linting failures.

Thanks for the input.  I will look into your suggestions.